public/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>fileFU  |  HOME</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    
+    <div class="container">
+      <div class="row">
+        <div class ="col-md-6"><p><h1 class="text-center">fileFU</h1></div>
+      </div>
+
+      <div class="row">
+        <div class ="col-md-6"><p class="text-center">fileFU is a simple file server brought to you by Dragonegger2.</div>
+      </div>
+
+        <form method="post" action="/api/upload", enctype="multipart/form-data" style="padding-top:10">
+          <div class="row text-center">
+            <div class="col-md-6">
+              <div class="input-group">
+                <span class="input-group-btn">
+                    <span class="btn btn-primary btn-file">
+                        Browse&hellip; <input type="file" multiple name="image">
+                    </span>
+                  </span>
+                <input type="text" class="form-control" readonly>
+              </div>
+            </div>
+          </div>
+          <div class="row text-center" style="padding-top:10px;">
+            <div class ="col-md-6">
+              <button type="submit" class="btn btn-primary">
+                Upload
+              </button>
+            </div>
+          </div>
+        </form>
+      <div class="row">
+        <div id="download" class="col-md-6">
+          <table class ="table">
+            <thead>
+            <th>File Name</th>
+            <th>Download</th>
+            </thead>
+            <tbody>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      </div>
+
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script src="/javascript/global.js"></script>
```

Comment:
Without this ref, the table does not seem to populate; my assumption -- initial ref handles the alert on page load while this ref populates the table once the page has drawn (it's sitting right before the </body>)